pir/fast_pir/fast_pir_client.hpp

```diff
@@ -165,16 +161,19 @@ class FastPIRClient {
   }
 
  private:
+  // because we "batch" PIR encryption together, we need to know the keypair
```

Comment:
let's move these comments to the line above pir_map

Reply:
resolved.